rosidl_typesupport_connext_c/cmake/rosidl_typesupport_connext_c_generate_interfaces.cmake

```diff
@@ -267,3 +257,30 @@ if(NOT rosidl_generate_interfaces_SKIP_INSTALL)
 
   ament_export_libraries(${rosidl_generate_interfaces_TARGET}${_target_suffix} ${Connext_LIBRARIES})
 endif()
+
+if(NOT "${_generated_msg_files}${_generated_srv_files} " STREQUAL " ")
+  find_package(ament_cmake_cppcheck)
```

Comment:
yeah I agree. Have noticed these warnings http://ci.ros2.org/job/ci_linux/1323/consoleFull#console-section-84 which could have come from https://github.com/ros2/rosidl/commit/454d8d5c70ed54670efe8922d4bd4243f1b0e47a that's already merged